.github/workflows/lint-codeql.yaml

```diff
@@ -1,8 +1,10 @@
 name: codeql
 
 on:
-  push: {}
   pull_request: {}
+  push:
+    branches:
+      - master
```

Comment:
It will need to run also for pushes to the stable branches. Basically, CodeQL seems to expect, for each PR, to have a base run, that is a CodeQL run for the commit the PR is based on. If it's not the case, it complains. That's what b08f700 ("workflows: Skip jobs instead of workflows") was trying to fix.

Reply:
Right, that will be done in each particular backport.

Reply:
If you do it here, you skip the need to remind each backporter about it. Less chances of an oversight ;)

Reply:
I'm the backporter this week :smile:

Reply:
They do work, we have these ones for example https://github.com/cilium/cilium/blob/v1.10/.github/workflows/build-images-hotfixes.yaml#L6